doc/rados/configuration/pool-pg.conf

```diff
@@ -10,12 +10,13 @@
 	osd_pool_default_size = 3  # Write an object 3 times.
 	osd_pool_default_min_size = 2 # Accept an I/O operation to a PG that has two copies of an object.
 
+	# Disable autoscaling for manual management of placement groups.
+	osd_pool_default_pg_autoscale_mode = off
+
 	# Ensure you have a realistic number of placement groups. We recommend
 	# approximately 100 per OSD. E.g., total number of OSDs multiplied by 100
 	# divided by the number of replicas (i.e., osd pool default size). So for
 	# 10 OSDs and osd pool default size = 4, we'd recommend approximately
 	# (100 * 10) / 4 = 250.
         # always use the nearest power of 2
-
```

Comment:
The first `3` above should be a `three`, would be groovy to amend that in since you're already in this file.

Reply:
If I understand correctly, you are referring to the rule that lower numbers should be spelled out in sentences, right? Changed two instances of that and some other minor consistency issues in that comment, let me know if that's not what you wanted

Reply:
Yes, I think the rule is any number under 10 / ten but there are probably variants.  Yes this is nitpicky.